app/models/team.rb

```diff
@@ -0,0 +1,4 @@
+class Team < ApplicationRecord
+  validates :name, length: { minimum: 4, maximum: 50 }
```

Comment:
In my opinion, this should be a (candidate) key -- if not, something else has to be (a "`team_code`" or "`handler`"? let's discuss.)

[rein gem](https://github.com/nullobject/rein) is already installed in the project, so, when the above is decided, let's map the unique constraint both in the DBMS and in the app layer.

Also, I don't know if there is such a thing, but can these constraints (# of chars in the string) be mapped to the RDBMS schema?